### PR TITLE
Split runtime jobs based on Debian flavor

### PR DESCRIPTION
**Pull request by cormacpayne**
- [ ] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as \#\<issue\_number\>.
- [ ] Tests are included and/or updated for code changes.
- [ ] Proper license headers are included in each file.

### CODE REVIEW

File: tests/Oryx.RuntimeImage.Tests/DotNetCore/DotnetCoreImageVersionsTest.cs [hunk 1]
```diff
@@ -97,21 +99,46 @@ public void DotNetCoreBusterRuntimeImage_Contains_VersionAndCommit_Information(s
...
             var result = _dockerCli.Run(new DockerRunArguments
             {
-                ImageId = _imageHelper.GetRuntimeImage("dotnetcore", version, osType),
+                ImageId = _imageHelper.GetRuntimeImage("dotnetcore", version, ImageTestHelperConstants.OsTypeDebianBuster),
+                CommandToExecuteOnRun = "dotnet",
+                CommandArguments = new[] { "--info" }
+            });
+
+            // Assert
+            var actualOutput = string.Join("", result.StdOut.ReplaceNewLine().Where(c => !char.IsWhiteSpace(c)));
```
Comment (daniv-msft) [inline tests/Oryx.RuntimeImage.Tests/DotNetCore/DotnetCoreImageVersionsTest.cs:120]: I know we want this available quickly so I'm not suggesting that we fix all this tonight, but would we see any alternatives in the future to us duplicating a lot of this code? Would helper functions be a path to reduce code duplication?

**Review by daniv-msft — DISMISSED** [review]

File: tests/Oryx.RuntimeImage.Tests/DotNetCore/DotnetCoreImageVersionsTest.cs (hunk 1 above)
Comment (cormacpayne) [reply tests/Oryx.RuntimeImage.Tests/DotNetCore/DotnetCoreImageVersionsTest.cs:120]: Yep we can definitely refactor this once things have settled -- we have examples of doing this across other places from previous refactors, so it's just a matter of finding some time to do it.

Comment (cormacpayne) [reply tests/Oryx.RuntimeImage.Tests/DotNetCore/DotnetCoreImageVersionsTest.cs:120]: In these cases, the only thing that's different per test is the Debian flavor that we're using, so we could just turn the guts of these tests into a common private method that is then called from the tests themselves, and the tests will provide the Debian flavor they want to us (along with the other arguments they're already providing, like platform version).

**Review by daniv-msft — APPROVED** [review]

**Comment by waliMSFT** [comment]
need to merge this PR asap. `No disk space` is blocking the pipelines now.

**Pull request merged**